packages/project-utils/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@webiny/project-utils",
+  "telemetryVersion": "1.0.1",
```

Comment:
Can you describe this a bit to me?

Reply:
So I wanted a place to store the _local_ telemetry version number so I can compare it to the remote version so thought it best to store it in the `package.json`. Does that make sense or were you thinking of storing it somewhere else?